sensu/plugins/metrics-network-queues.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+#
+#  metrics-network-queues.py
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#  Python 2.7+ (untested on Python3, should work though)
+#
+# USAGE:
+#
+# metrics-network-queues.py -n <process_name> [-s <graphite_scheme>] [-z <criticality>]
+#
+# DESCRIPTION:
+# Finds the pid[s] corresponding to a process name and obtains the lengths of
+# the receive and send network queues for all of the process' sockets. This
+# data is gathered from the "netstat -tpane" command.
+#
+# Code adapted from Jaime Gogo's script in the Sensu Plugins community
+# https://github.com/sensu-plugins/sensu-plugins-process-checks/blob/master/bin/metrics-per-process.py
+#
+# Released under the same terms as Sensu (the MIT license); see MITLICENSE
+# for details.
+#
+# Siva Mullapudi <scmullap@us.ibm.com>
+
+import argparse
+import sys
+import os
+import time
+import subprocess
+from collections import Counter
+
+STATE_OK = 0
+STATE_WARNING = 1
+STATE_CRITICAL = 2
+CRITICALITY = 'critical'
```

Comment:
this is only needed for checks ... metrics do not need this as they do not create alerts.